### PR TITLE
WINDUPRULE-460 getProperties and setProperty methods have been remove…

### PR DESCRIPTION
**Pull request by johnpoth**
…d from CamelContext

https://issues.redhat.com/browse/WINDUPRULE-460

To run tests: `mvn -Dtest=WindupRulesMultipleTests -DrunTestsMatching=java-generic-information clean surefire-report:report`

Thanks !

### CODE REVIEW

**Comment by m-brophy** [comment]
retest

**Comment by m-brophy** [comment]
@johnpoth can you merge master into this PR please?  I'm led to believe this will fix the failing tests.

**Comment by johnpoth** [comment]
@m-brophy done, rebased and pushed, thanks !

**Comment by m-brophy** [comment]
thanks @johnpoth , reviewing now!

File: rules-reviewed/camel3/camel2/tests/data/java-generic-information/camelContextDeprecatedMethodsBlueprint.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+
+
+<camelContext id="camel1" xmlns="http://camel.apache.org/schema/spring">
```
Comment (m-brophy) [inline rules-reviewed/camel3/camel2/tests/data/java-generic-information/camelContextDeprecatedMethodsBlueprint.xml]: I think this should be the blueprint schema

Comment (johnpoth) [reply rules-reviewed/camel3/camel2/tests/data/java-generic-information/camelContextDeprecatedMethodsBlueprint.xml]: @m-brophy fixed, thanks !

**Pull request merged**